pkg/util/cloudinit/validate/validate_test.go

```diff
@@ -97,6 +97,24 @@ var _ = Describe("Validate CloudConfigJSONRawMessage", func() {
 			})
 		})
 	})
+
+	When("The CloudConfig's first file content is unset", func() {
+		BeforeEach(func() {
+			cloudConfig.WriteFiles[0].Content = nil
+		})
+		It("Should not return any errors", func() {
+			Expect(errs).To(HaveLen(0))
+		})
+	})
+
+	When("The CloudConfig's first file content is empty", func() {
+		BeforeEach(func() {
+			cloudConfig.WriteFiles[0].Content = []byte(``)
```

Comment:
Also please have a test where this field is `nil` and not `[]byte(``)`

Reply:
https://github.com/vmware-tanzu/vm-operator/pull/334/files#diff-71149d5c5d525b6bff3a89a726ea2fd1bc114febde84de2b25bb4feb79012ba2R103

Are you thinking of something other than the one added?

Reply:
Somehow I missed that, thanks!